kong/plugins/prometheus/exporter.lua

```diff
@@ -141,6 +141,17 @@ local function init()
                                                      {"node_id", "shared_dict", "kong_subsystem"},
                                                      prometheus.LOCAL_STORAGE)
 
+  if kong.configuration.database == "off" then
+    memory_stats.lmdb = prometheus:gauge("memory_lmdb_used_bytes",
```

Comment:
```suggestion
    memory_stats.lmdb = prometheus:gauge("lmdb_mmap_used_bytes",
```

Reply:
I think `mmap` style metrics name is not good, Kong may have many mmap-based features in the future, but here only LMDB-related metrics are involved, so changing the name to mmap does not make sense.

Reply:
I agree with @attenuation , `mmap` is too technical.

Reply:
let's se `lmdb_memory_used_bytes` then, having the context at front is closer to prometheus naming convention; same applies to other metrics